docs/api/ror-internal-api-swagger.yaml

```diff
@@ -2,7 +2,7 @@ openapi: 3.0.3
 info:
   title: ROR Internal API
   description: The API is designed for ROR KBN plugin. It should be treated as internal. It means that backward compatibility won't be a first-class thing for us.
-  version: 2.1.0
+  version: 3.0.0
```

Comment:
maybe we will introduce a changelog for the API? It'll be easier for the Frontend team to follow the changes. WDYT?